### ISSUE TITLE
Incorrect handling of multiple classes with same package and name (in different modules)

### ISSUE THREAD

**Issue by michalkochanowicz** [andrepdo/findbugs-idea#115]
I have multiple branches of same project in Idea imported as separate modules.

So I have multiple versions of same classes (with same package names).

FindBugs seems to be trying to open class from wrong Idea module. The plugin should check which module was actually analysed or at least ask user from which module open the class.

This is how analyse stacktrace window acts.

Exception occurred when I clicked on problem in left pane.

```
Wrong line: 456. Available lines count: 313
java.lang.IndexOutOfBoundsException: Wrong line: 456. Available lines count: 313
	at com.intellij.openapi.editor.impl.LineSet.checkLineIndex(LineSet.java:169)
	at com.intellij.openapi.editor.impl.LineSet.getLineStart(LineSet.java:154)
	at com.intellij.openapi.editor.impl.DocumentImpl.getLineStartOffset(DocumentImpl.java:926)
	at org.twodividedbyzero.idea.findbugs.gui.toolwindow.view.BugTreePanel.createEditor(BugTreePanel.java:241)
	at org.twodividedbyzero.idea.findbugs.gui.toolwindow.view.BugTreePanel.setPreview(BugTreePanel.java:197)
	at org.twodividedbyzero.idea.findbugs.gui.tree.view.BugTree$SelectionListenerImpl.valueChanged(BugTree.java:343)
	at javax.swing.JTree.fireValueChanged(JTree.java:2927)
	at javax.swing.JTree$TreeSelectionRedirector.valueChanged(JTree.java:3391)
	at javax.swing.tree.DefaultTreeSelectionModel.fireValueChanged(DefaultTreeSelectionModel.java:635)
	at javax.swing.tree.DefaultTreeSelectionModel.notifyPathChange(DefaultTreeSelectionModel.java:1093)
	at javax.swing.tree.DefaultTreeSelectionModel.setSelectionPaths(DefaultTreeSelectionModel.java:294)
	at javax.swing.tree.DefaultTreeSelectionModel.setSelectionPath(DefaultTreeSelectionModel.java:188)
	at javax.swing.JTree.setSelectionPath(JTree.java:1634)
	at javax.swing.plaf.basic.BasicTreeUI.selectPathForEvent(BasicTreeUI.java:2393)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.handleSelection(BasicTreeUI.java:3609)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.mousePressed(BasicTreeUI.java:3548)
	at com.intellij.util.ui.MouseEventAdapter.mousePressed(MouseEventAdapter.java:42)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6532)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6300)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4891)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4522)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:857)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:654)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

**Comment by michalkochanowicz** [comment]
There is a similar problem with preview window. It shows source from incorrect module (same class and package name).

**Comment by ccsplit** [comment]
I am also seeing the preview behavior and when clicking on the bug it will go to the incorrect module, with the following information:
```
Mac OS X: 10.11.4
IntelliJ IDEA 2016.1.2
Build #IC-145.972, built on May 14, 2016
JRE: 1.8.0_76-release-b162 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Findbugs-IDEA: 0.9.997
```

**Comment by retomerz** [comment]
Fixed in v0.9.998.
